src/Dfe.PlanTech.DatabaseUpgrader/Scripts/2024/20240726_1201_AddCSUrlToRecommendationChunk.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE Contentful.RecommendationChunk
+    ADD [CSURL] NVARCHAR(256)
```

Comment:
This'll be the one! missing an `s` off RecommendationChunk
I suggest you combine this with the other sql script so we just have the one and you can add both columns together with 
```sql
ALTER TABLE Contentful.RecommendationChunks
    ADD [CSUrl] NVARCHAR(256), [CSLinkText] NVARCHAR(256)
```

Reply:
[e6fe08d](https://github.com/DFE-Digital/sts-plan-technology-for-your-school/pull/718/commits/e6fe08d4278b00680e916db5ecaef1a3f582c732)

Reply:
[a9f8581](https://github.com/DFE-Digital/sts-plan-technology-for-your-school/pull/718/commits/a9f85815993b92586abdd9111f0e5320c9c50ed3)